code/modules/cargo/exports/sheets.dm

```diff
@@ -28,13 +27,13 @@
 
 /datum/export/stack/skin/cat
```

Comment:
Does this actually still exist?
From cats, or felinids?

Reply:
I'm not sure if it's obtainable, but the item does still exist in the code.  Maybe you can skin Runtime somehow?  I'm not sure.  Probably better to leave it here just in case until/unless someone specifically removes the item, I think.